BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -975,3 +975,6 @@
 "PleaseRestartRegistrationOrTryLoggingInYouMayAlreadyHaveAnAccount" = "Please restart registration or try logging in. You may already have an account.";
 "RemovePasskey" = "Remove passkey";
 "PasskeyRemoved" = "Passkey removed";
+"SetUpLaterQuestion" = "Set up later?";
+"YouCanFinishSetupUnlockAnytime" = "You can finish setup anytime in Account Security under Settings. You’ll use your master password to unlock until you set up another method.";
```

Comment:
⛏️ Could the suffix `DescriptionLong` be added to be alike the other really long strings?

`YouCanFinishSetupUnlockAnytimeDescriptionLong`

Reply:
Yep!